registry/navigationMenuItems.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE groups SYSTEM "../lib/pkp/dtd/navigationMenus.dtd">
+
+<!--
+  * registry/navigationMenuItems.xml
+  *
+  * Copyright (c) 2014-2017 Simon Fraser University
+  * Copyright (c) 2003-2017 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Default NavigationMenuItems XML file.
+  *
+  -->
+
+<navigationMenuItems>
+	<!-- User -->
+	<navigationMenuItem title="user.logOut" op="signOut" page="login" default="1" default_id="8">
+		<navigationMenuItem title="navigation.admin" op="index" page="admin" default="1" default_id="10" />
+		<navigationMenuItem title="common.viewProfile" op="profile" page="user" default="1" default_id="11" />
```

Comment:
I think my comments elsewhere suggest removing the IDs in favour of symbolic names -- but whenever you do put a magic number into e.g. a config file, make sure you add the symbolic name there in comments too e.g. `<!-- MY_CONSTANT_NAME_HERE -->` so that grepping for the constant will turn up hits here too.